### ISSUE TITLE
Map compute (nova) events based on what openstack returns to ceilometer

### ISSUE THREAD

**Issue by rob-baron** [CCI-MOC/xdmod-cntr#16]
Here is a list of compute events that are currently passed to xdmod:

compute.instance.create.error
compute.instance.create.start
compute.instance.shutdown.end
compute.instance.shutdown.start
compute.instance.create.end
compute.instance.delete.start
compute.instance.live_migration._post.end
compute.instance.live_migration._post.start
compute.instance.live_migration.post.dest.end
compute.instance.live_migration.post.dest.start
compute.instance.live_migration.pre.end
compute.instance.live_migration.pre.start
compute.instance.power_off.end
compute.instance.power_off.start
compute.instance.power_on.end
compute.instance.power_on.start
compute.instance.shutdown.end
compute.instance.shutdown.start
compute.instance.exists
compute.instance.volume.attach
compute.instance.volume.detach

**Comment by joachimweyl** [comment]
This is required for a minimum viable product.

**Comment by rob-baron** [comment]
So I managed to have xdmod display data using the compute.create and compute.exists events.

data only appears in the interface after it is ingested.

some data will appear with just the compute.exsits events. - not sure how many of these are required.  ceilometer pumped these out at a rate of 1 per hour.

Unfortunately, the data disappeared when I went to demo it.  What is more confusing is that I didn't run the shredding or the ingesting scripts.  What this indicates to me is that I don't fully understand what data xdmod requires to be consistent. 

However, this is not the case for the reference data.  So I need to further analyze the reference data in order to understand which fields need to be consistent across events.

I suspect part of this is due to the create.start and create.end events having the same generation time.

Furthermore, I have no idea what the proper ordering of live_migration events, or if there is some context that I don't understand at this point.

**Comment by rob-baron** [comment]
Also, need to work at making compact files - less than 5000 events.  This is so that processing can occur in a reasonable amount of time.

**Comment by joachimweyl** [comment]
@rob-baron when it comes to the data disappearing, can we open a ticket with XDMoD, if we don't already have one?

**Comment by rob-baron** [comment]
I think it is due to me making a bad assumption about the data they are using.  I did ask about the audit period as that was an obvious one.  The responses so far have been what data isn't important, not what data is.  Furthermore, some of the data has changed - and I have submitted queries concerning those changes.

In order to get a more focused question, I have reformatted all of the compute event data into a transaction like log for this issue (https://github.com/CCI-MOC/xdmod-cntr/issues/28)
